ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -169,6 +137,26 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     @objc public static var mediumHeight: CGFloat { return height(title: "", subtitle: " ") }
     @objc public static var largeHeight: CGFloat { return height(title: "", subtitle: " ", footer: " ") }
 
+    /// TableViewCell colors with obj-c support
+    @objc public static func tableBackgroundColor(fluentTheme: FluentTheme) -> UIColor {
+        UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.background1])
+    }
+    @objc public static func tableBackgroundGroupedColor(fluentTheme: FluentTheme) -> UIColor {
+        UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.canvasBackground])
+    }
+    @objc public static func tableCellBackgroundColor(fluentTheme: FluentTheme) -> UIColor {
+        UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.background1])
```

Comment:
i would have thought you will be referencing to tableviewtokens.swift itself not referencing fluenttheme.aliastoken.